WordPress/Classes/Utility/Ratings/AppRatingsUtility.swift

```diff
@@ -18,6 +18,16 @@ class AppRatingUtility: NSObject {
     ///
     var appReviewUrl: URL = Constants.defaultAppReviewURL
 
+    /// Don't prompt for reviews for internal builds
+    /// http://stackoverflow.com/questions/27297435/detect-if-ios-app-is-downloaded-from-apples-testflight
+    ///
+    var isInternalBuild: Bool = {
+        guard let path = Bundle.main.appStoreReceiptURL?.path else {
```

Comment:
Also, I'm not sure how to test it, but `appStoreReceiptURL` might not be present for `WP Internal` builds